.travis.yml

```diff
@@ -1,6 +1,7 @@
 language: node_js
 node_js:
-  - '8.9.4'
+  - 8
+  - 10
```

Comment:
@DanielRuf  Thanks for making this PR.  Do you know when NodeJS 10.x will transition to Long Term Support (LTS)?  We usually don't try to support non-LTS versions because of past issues with bugs.  If it's right around the corner, then I'm okay with merging.

Reply:
See https://github.com/nodejs/Release/blob/master/README.md